#424 Update Requirements (#426)
Inbibo/adonisfx_docs: docs/requirements.md

- **OS**: Microsoft Windows 10+, Linux CentOS 7+, Linux Rocky 8+, Linux RHEL 8+ (64-bit).
- **CPU**: 64-bit Intel or AMD multi-core processor.

> - **GPU**: NVIDIA GPU required for GPU-accelerated ML workflows, with 12GB of VRAM or more recommended. CPU fallback is supported when GPU execution is not available, but it can be slower.

Reviewer: ```suggestion
- **GPU**: Nvidia GPU suggested for accelerated ML workflows, with 6GB of VRAM or more. CPU fallback is supported when GPU execution is not available, but it can be slower.
```

Author: Done!

- **RAM**: 8GB required (16GB or more recommended).

> - **Storage**: At least 700MB of free disk space to install. Installing the ML dependencies requires approximately 6GB to 7GB of free disk space.

Reviewer 2: ```suggestion
- **Storage**: At least 700MB of free disk space to standard install. If ML dependencies are required, then at least 7GB of free disk space.
```

Author: Done

> [!NOTE]
> - Adonis may work with other CPU, GPU, RAM, and VRAM specifications.
> - The list above represents the specifications that have been tested successfully.

> > - Internet connection is required for licensing only.

Reviewer: ```suggestion
> - Internet connection is required for licensing and ML dependencies download only.
```

Author: Done!

> > - Standalone script execution outside the DCC requires a separate Python interpreter to be installed.

Reviewer 2: The standalone interpreter is part of the package, let's remove this line because it may sound like they have to take care of installing.

```suggestion

```

Author: Done

> - ML features run locally and by default do not require cloud services. Training data remains on the local machine by default.

> > - FX users can install the ML dependencies to use GPU inference, but this is optional.

Reviewer: ```suggestion
> - FX Windows users can install the ML dependencies to use GPU inference, but this is optional.
```

Author: Done!

> - Some features require a specific Adonis license bundle. See [Licensing](licensing.md) for details. For example, FX features require an AdonisFX bundle, and ML features require an AdonisML bundle.
> - Generating ML training data and training models may require additional disk space and memory depending on the dataset and model size.

> > - Trained models are generally expected to be under 2GB, but extracted training data may exceed 10GB depending on the scenario.

Reviewer 2: Let's not add everything in the note. 
Maybe we could review the sections and have:

```
# Requirements

## Software

foo

## System

foo

## AdonisFX

- about installing dendencies for GPU inferece

## AdonisML

- About specific license
- about model size
- about training data 
```

Author: Done

- **OS**: Microsoft Windows 10+, Linux CentOS 7+, Linux Rocky 8+, Linux RHEL 8+ (64-bit).
- **CPU**: 64-bit Intel or AMD multi-core processor.
- **GPU**: NVIDIA GPU suggested for accelerated ML workflows, with 6GB of VRAM or more recommended. CPU fallback is supported when GPU execution is not available, but it can be slower.
- **RAM**: 8GB required (16GB or more recommended).
- **Storage**: At least 700MB of free disk space for the standard install. If ML dependencies are required, then at least 7GB of free disk space.

> [!NOTE]
> Adonis may work with other CPU, GPU, RAM, and VRAM specifications. The list above represents the specifications that have been tested successfully.

> [!NOTE]
> Internet connection is required for licensing and ML dependencies download only.

## AdonisFX

Some AdonisFX features require an AdonisFX license bundle. See [Licensing](licensing.md) for details.

> FX Windows users can install the ML dependencies to use GPU inference, but this is optional.

Reviewer 2: I suggest to remove the first one, and reformulate the second one.

```suggestion
## AdonisFX

- With an Adonis FX license (see [licensing](licensing.md) for details) installing the ML dependencies is optional. It is only required to run ML inference on the GPU. If not installed, the ML inference runs on the CPU.
```

Author: Done!

> ## AdonisML

Reviewer 2: I would follo the formatting using a list. I agree with the content

```suggestion
## AdonisML

- ML features require an Adonis ML license bundle. See [Licensing](licensing.md) for details.
- ML features run locally and do not require cloud services. Training data remains on the local machine.
- Generating ML training data and training models may require additional disk space and memory depending on the dataset and model size.
- Trained models are generally expected to be under 2GB, but extracted training data may exceed 10GB depending on the scenario.
```

Author: Done!

AdonisML features require an AdonisML license bundle. See [Licensing](licensing.md) for details.

> ML features run locally and by default do not require cloud services. Training data remains on the local machine by default.

Reviewer 3: Optional suggestion. I say this because when saying "by default" it may give the impression that you can change options to run ML features on the cloud and send training data somewhere.
```suggestion
ML features run locally do not require cloud services. Training data remains on the local machine.
```

Author: Done!

> - Adonis may work with other CPU, GPU, RAM, and VRAM specifications.
> - The list above represents the specifications that have been tested successfully.
> - Internet connection is required for licensing only.

> > - Standalone script execution outside Maya or Houdini requires a separate Python interpreter to be installed.

Author: Use DCC not Maya and Houdini
